spec/queries/get_fraud_matches_spec.rb

```diff
@@ -75,5 +75,21 @@
         expect(postcodes).to include('W69BH')
       end
     end
+
+    context 'when duplicated unsubmitted applications' do
```

Comment:
Could this whole test be combined with the one above, by setting `submitted_at: nil` on one or both of those? (For the sake of test speed?)

Reply:
I am not sure. It is a different context and a different scenario, I'll feel safer and confident that everything works, with more tests for different scenarios even if adds more time for the tests to run.